mottak-arkiv-service/app/jobs/services/workflow_metadata_services.py

```diff
@@ -111,3 +129,50 @@ async def save_workflow_metadata(metadata_response: WorkflowMetadataResponse, db
 
     if metadata_response.workflow_type == WorkflowType.TRANSFER_TO_BEVARING:
         await _save_workflow_of_type_transfer_to_bevaring(metadata_response, db)
+
+
+async def _get_arkivuttrekk_id_by_overforingspakke_id(overforingspakke_id: str, db: AsyncSession) -> Optional[int]:
+    arkivuttrekk = await arkivuttrekk_repository.get_by_overforingspakke_id(db, int(overforingspakke_id))
+    return arkivuttrekk.id
+
+
+async def _update_arkivuttrekk_status(db: AsyncSession, arkivuttrekk_id: int, workflow_status: str) -> None:
```

Comment:
Blir ikke dette korrekt?
```suggestion
async def _update_arkivuttrekk_status(db: AsyncSession, arkivuttrekk_id: int, workflow_status: WorkflowStatus) -> None:
```

Reply:
Jo, men ser at jeg har gjort en feil når jeg oppdatere status i `_save_workflow_of_type_transfer_to_bevaring` tidligere. Ny kode er pushet som inneholder denne suggestion.